CSS/components/modal.css

```diff
@@ -0,0 +1,39 @@
+.dialog-main{
+    display: none;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+    width: 100vw;
+    height: 100vh;
+}
+.open{
+    display: flex;
+}
+.dailog-btn-wrap{
+    display: flex;
+    justify-content: flex-end;
+    gap: 1rem;
+}
+.dialog-btn{
+    display: flex;
+    justify-content: flex-end;
+    border: none;
+    width: 5rem;
+    height: 2.5rem;
+    cursor: pointer;
+    border-radius: 1rem;
+}
+.dialog{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 1rem;
+    width: 15rem;
+    z-index: 2;
+    max-height: 15rem;
```

Comment:
Reconsider height and width, the content might exceed one line. It may overflow in that case.

Reply:
Thank you for pointing out did the height: fit content , Now it's not overflowing
![Screenshot (423)](https://user-images.githubusercontent.com/75560307/152894015-0df705bb-c41a-4e74-99c6-aa51dbd30a9d.png)